src/main/java/de/mediathekview/mserver/crawler/rbb/tasks/RbbFilmTask.java

```diff
@@ -0,0 +1,104 @@
+package de.mediathekview.mserver.crawler.rbb.tasks;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import de.mediathekview.mlib.daten.Film;
+import de.mediathekview.mlib.daten.Resolution;
+import de.mediathekview.mlib.daten.Sender;
+import de.mediathekview.mserver.crawler.ard.json.ArdVideoInfoDTO;
+import de.mediathekview.mserver.crawler.ard.json.ArdVideoInfoJsonDeserializer;
+import de.mediathekview.mserver.crawler.basic.AbstractCrawler;
+import de.mediathekview.mserver.crawler.basic.AbstractDocumentTask;
+import de.mediathekview.mserver.crawler.basic.AbstractRecrusivConverterTask;
+import de.mediathekview.mserver.crawler.basic.CrawlerUrlDTO;
+import de.mediathekview.mserver.crawler.rbb.parser.RbbFilmDetailDeserializer;
+import de.mediathekview.mserver.crawler.rbb.parser.RbbFilmInfoDto;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentLinkedQueue;
+import mServer.crawler.CrawlerTool;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.jsoup.nodes.Document;
+
+public class RbbFilmTask extends AbstractDocumentTask<Film, CrawlerUrlDTO> {
+
+  private static final Logger LOG = LogManager.getLogger(RbbFilmTask.class);
+
+  private final RbbFilmDetailDeserializer filmDetailDeserializer;
+  private final Gson gson;
+
+  public RbbFilmTask(AbstractCrawler aCrawler,
+      ConcurrentLinkedQueue<CrawlerUrlDTO> aUrlToCrawlDtos) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The type of the "aUrlToCrawlDtos" object should be an interface such as "Queue" rather than the implementation "ConcurrentLinkedQueue". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)